.github/workflows/ci.yml

```diff
@@ -58,6 +59,23 @@ jobs:
         run: npm ci --ignore-scripts
       - name: Build
         run: npm run docs
+      - name: Paths Changes Filter
+        uses: dorny/paths-filter@v2.11.1
+        id: changes
+        with:
+          filters: |
+            puppeteer:
+              - 'packages/puppeteer/**'
+              - 'packages/puppeteer-core/**'
+              - 'docker/**'
+              - 'test/**'
+              - 'test-d/**'
+              - 'tools/mochaRunner/**'
+            website:
+              - 'docs/**'
+              - 'website/**'
+            ng-schematics:
+              - 'packages/ng-schematics/**'
```

Comment:
I think we should trigger all jobs if workflows change (to make sure things still work after the workflow changes)